test/sol6/kyberDao.js

```diff
@@ -3586,35 +3585,6 @@ contract('KyberDAO', function(accounts) {
       await feeHandler.withdrawAllETH({from: accounts[0]});
     });
 
-    it("Test claim reward should revert fee handler return false for claimReward", async function() {
```

Comment:
Should still keep this test, but change to: "Test claim reward should revert if fee handler failed to claim reward` and make it revert in FeeHandler `claimStakerReward` function